File: snippets/dart_snippets/lib/production.dart
import 'dart:typed_data';

import 'package:breez_sdk/bridge_generated.dart';

NodeConfig productionNodeConfig() {
  // ANCHOR: moving-to-production
  // Read in your Greenlight credentials from the file 
  // system, environment variable or build config
  Uint8List developerKey = Uint8List(0);
  Uint8List developerCert = Uint8List(0);
  GreenlightCredentials greenlightCredentials = GreenlightCredentials(
    developerKey: developerKey, 
    developerCert: developerCert,
  );

  NodeConfig nodeConfig = NodeConfig.greenlight(
    config: GreenlightNodeConfig(
      partnerCredentials: greenlightCredentials,
      inviteCode: null,
    ),
  );
  // ANCHOR_END: moving-to-production
  return nodeConfig;
}

File: snippets/swift/BreezSDKExamples/Sources/Production.swift
//
//  Production.swift
//
//
//  Created by dangeross on 29/11/2023.
//

import BreezSDK

func productionNodeConfig() -> NodeConfig {
    // ANCHOR: moving-to-production
    // Read in your Greenlight credentials from the file 
    // system, environment variable or build config
    let developerKey = [UInt8]()
    let developerCert = [UInt8]()
    let greenlightCredentials = GreenlightCredentials(developerKey: developerKey, developerCert: developerCert)

    let nodeConfig = NodeConfig.greenlight(
        config: GreenlightNodeConfig(partnerCredentials: greenlightCredentials, inviteCode: nil))
    // ANCHOR_END: moving-to-production
    return nodeConfig
}
